backend/kernelCI_app/queries/build.py

```diff
@@ -52,8 +52,8 @@ def get_build_details(build_id: str) -> Optional[list[dict]]:
         return query.select()
     except ProgrammingError as e:
         if is_valid_does_not_exist_exception(e):
-            set_schema_version(version="5")
-            log_message("Build Details -- Schema version updated to 5")
+            set_schema_version()
+            log_message("Build Details -- Schema version updated")
```

Comment:
nit: You could add the DEFAULT_SCHEMA_VERSION in this message. And the same goes to the other instances of `log_message`

```suggestion
            log_message(f"Build Details -- Schema version updated to {DEFAULT_SCHEMA_VERSION}")
```